data/adamas missions.txt

```diff
@@ -11,17 +11,17 @@
 mission "Adamas Wormhole Opens"
 	name "Investigate wormhole"
 	invisible
-	minor
 	description "Opens the Adamas wormhole - you shouldn't be seeing this."
 	waypoint "At Andol"
 	destination "Maker"
 	to offer
 		has "kestrel available"
 		"reputation: Syndicate" >= 1
 	on offer
-		event "at andol wormhole opens"
-		require "Jump Drive"
-		fail
+		event "andol wormhole opens"
+		event "small adamas outfitter chrysoprase" 70
+		event "dust wormhole opens" 30
+		event "cygni wisp opens" 25
```

Comment:
Seeing my comments elsewhere, I think "andol wormhole opens" is the only event that should apply here since:
1. Small Adamas Outfitter was added without an event.
2. Dust wormhole opens should be used later, during the storyline.
3. Cygni wisp is a super outdated event referencing another nonexistent system.

Reply:
Fixed in my fork, don't know if that applies to this PR also.

Reply:
It is fixed.

Reply:
Ok, so I have updated all the outdated outfits, so that is correct.
Are there any other files that should be changed?
If not, this PR is ready to be merged.